lib/Driver/ToolChains.cpp

```diff
@@ -119,6 +119,22 @@ static void addCommonFrontendArgs(const ToolChain &TC,
     arguments.push_back(inputArgs.MakeArgString(OI.SDKPath));
   }
 
+  // HACK: Add an -iquote Clang search path (using -Xcc) in order to cope with
+  // differences between precompiled and non-precompiled bridging headers.
+  //
+  // In non-PCH contexts, a module map in the same directory as the bridging
+  // header is read in by the Clang importer. To replicate that behavior with
+  // PCH, add an extra -iquote search path at the start of the search list.
+  // This won't affect any textual header resolution because the directory of
+  // the current file is already searched for quoted includes.
```

Comment:
Oof, of course. It may still be the best approach we've got but it needs more investigation. Thanks, Ben.